app/views/incomeSources/manage/ManageIncomeSourceDetails.scala.html

```diff
@@ -250,8 +250,10 @@
             ))
                 ++ getQuarterlyType
                 ++ {
-            if(viewModel.itsaHasMandatedOrVoluntaryStatusCurrentYear
-                    && viewModel.latencyDetails.isDefined) getTaxYears else Seq.empty
+            viewModel.latencyYearTwoIsQuarterly match {
+                case Some(true) if viewModel.latencyDetails.isDefined => getTaxYears
+                case _ => Seq.empty
+            }
```

Comment:
Could this be moved out of the view into the view model, just to remove a bit of logic from the view